server/apps/main/forms.py

```diff
@@ -50,6 +50,16 @@ class ShareExperienceForm(forms.Form):
     research = forms.BooleanField(label = "Share for research", required=False)
     research.group = 3
 
+    # authorship option
+    authorship = [
+        ("AutSPACEs contributor", "AutSPACEs contributor"),
+        ("Autistic individual", "Autistic individual"),
+        ("Parent of an autistic individual", "Parent of an autistic individual"),
+        ("Carer for an autisitc individual", "Carer for an autisitc individual"),
+    ]
+    experience_authorship = forms.ChoiceField(choices = authorship, widget = forms.Select(), required=False)
+    experience_authorship.group = 4
+
```

Comment:
I might be forgetting a discussion we had later, but I thought we had planned to go with a binary "Is this an experience shared on behalf of someone else?" and then Yes/No? 

And then having a second expansion, a free-text box which would record how/why the experience was submitted on behalf of someone else? 

At least that's the state we also have recorded in #544

Reply:
Yes we did look at that when starting and then kinda glossed over it!
Will look into implementation now

Reply:
Okay, no worries! I really wasn't sure what the latest outcome of our discussion was! 👍